API/server.js

```diff
@@ -28,19 +28,19 @@ function getRandomAnimalNo(){
 
 app.get('/', (req, res) => res.send('Hello World!'))
 
-app.get('/search/animal/:id', function(req, res){
+app.get('/search/animal/:id', function(req, res){ //
     // generate random number that equates to animal
-    getRandomAnimalNo();
+    getRandomAnimalNo(); //
 
     // send the random animal
     let animal = animals[number];
     if(!animal) {
-        res.send("Error: Please pick a number between 0 and 9");
+        res.send("Error: Please pick a number between 0 and 9"); //
```

Comment:
This line is never made use of, as the number passed to animal is generated server-side and so is always between 0 & 9 (as generated by `getRandomAnimalNumber();`